packages/building-webpack/loaders/import-meta-url-loader.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+const regex = /import\.meta/g;
+
```

Comment:
maybe have a separate package? 
how about `@open-wc/webpack-import-meta-loader`?

Reply:
yes better do the refactor of webpack => webpack-import-meta-loader later